app/views/events/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="container">
+  <div class="bg-info p-5 mt-3 mb-3">
+    <h1>Welcome to Private Events</h1>
+  </div>
+
+
+  <% if current_user %>
+    <%= link_to new_event_path do %>
+    <button class="border-0 bg-info text-light px-3 py-2 rounded mr-2">Create event</button>
+    <% end %>
+
+    <%= link_to current_user do %>
+    <button class="border-0 bg-info text-light px-3 py-2 rounded mr-2">Events i am attending</button>
+    <% end %>
+
+    <div class="mt-5">
+      <h3>Upcoming events</h3>
+      <div class="row">
+        <%= render partial: 'upcoming_event_card' %>
+      </div>
+      <h3 class="mt-3">Past Events</h3>
+      <div class="row">
+          <%= render partial: 'past_event_card' %>
+      </div>
+
+    </div>
+  <% end %>
```

Comment:
- [ ] use a helper method to render the if/end statement in this part of your code there should be no if/end statement within views if you have several lines you can use a partial to render the markup